File: guides/micronaut-data-mongodb-asynchronous/java/src/test/java/example/micronaut/MicronautguideTest.java
package example.micronaut;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Assertions;

class MicronautguideTest extends BaseMongoDataTest {

    @Test
    void testItWorks() {
        Assertions.assertTrue(application.isRunning());
    }

}

File: guides/micronaut-hashicorp-vault/kotlin/src/test/kotlin/example/micronaut/MicronautguideTest.kt
package example.micronaut

import io.micronaut.context.annotation.Property
import io.micronaut.runtime.EmbeddedApplication
import io.micronaut.test.extensions.junit5.annotation.MicronautTest
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import jakarta.inject.Inject
import org.junit.jupiter.api.Assertions.assertTrue

@MicronautTest
@Property(name = "vault.client.config.enabled", value = "false")
class MicronautguideTest {

    @Inject
    lateinit var application: EmbeddedApplication<*>

    @Test
    fun testItWorks() {
        assertTrue(application.isRunning)
    }

}
